assets/queries/openAPI/global_server_uses_http/query.rego

```diff
@@ -0,0 +1,52 @@
+package Cx
+
+import data.generic.openapi as openAPILib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openAPILib.checkOpenAPI(doc) != "undefined"
+    object.get(doc, "servers", "undefined") == "undefined"
```

Comment:
check indentation

Reply:
@felipe-avelar  this is some kind of Github illusion, click on the "view file" button on the top right of the diff to see that indentation is correct